docs/resources/appdata_dsource.md

```diff
@@ -0,0 +1,176 @@
+# Resource: <resource name> delphix_appdata_dsource
+
+In Delphix terminology, a dSource is a database that the Delphix Virtualization Engine uses to create and update virtual copies of your database. 
```

Comment:
Update to "Delphix Continuous Data Engine". 

Avoid "Delphix Virtualization Engine"